Replacing 'Solutions Site' with 'Admin Site' in all current scripting…

ashleygagregory opened the pull request:
…, templates and documentation

ashleygagregory committed:
Replacing 'Solutions Site' with 'Admin Site' in all current scripting, templates and documentation

Coianiz reviewed (changes requested):
Can we make it Administration Site to marry up with the client
![image](https://user-images.githubusercontent.com/60905116/215371826-0175d580-d164-4214-b5d5-467e0d756f2b.png)

ashleygagregory commented:
Should Site Title be 'OnePlace Solutions Administration Site' then? 

Looks a little weird being 'OnePlace Administration Site' and OnePlace is a different product.
![image](https://user-images.githubusercontent.com/56142786/215378784-91806603-a9ce-4de1-aa74-beb59acecbe3.png)

ashleygagregory committed:
Wording changes

Updating 'Admin Site' to 'Administration Site'
Updating 'OnePlace Administration Site' to 'OnePlace Solutions Administration Site' (OnePlace is a different product)

Coianiz commented:
> Should Site Title be 'OnePlace Solutions Administration Site' then?
> 
> Looks a little weird being 'OnePlace Administration Site' and OnePlace is a different product. ![image](https://user-images.githubusercontent.com/56142786/215378784-91806603-a9ce-4de1-aa74-beb59acecbe3.png)

I think it's ok as it is
